src/components/project-detail/HeaderSection.tsx

```diff
@@ -1,26 +1,23 @@
-import Image from 'next/image';
 import { css } from '@emotion/react';
 
 import { PROJECTS_IMAGE_BASE } from '~/constants/images';
 import { mediaQuery } from '~/styles/constants';
 
 import { Project } from '../project/constants';
 
-export default function HeaderSection({ image, title }: Project) {
+export default function HeaderSection({ image }: Project) {
   return (
     <section css={sectionCss}>
-      <Image
-        src={`${PROJECTS_IMAGE_BASE}/${image}`}
-        alt={title}
-        fill
-        priority
-        quality={100}
-        placeholder="blur"
-        blurDataURL={`${PROJECTS_IMAGE_BASE}/${image}`}
-        css={css`
-          object-fit: cover;
-        `}
-      />
+      <div
+        style={{
+          width: '100%',
+          height: '100%',
+          backgroundImage: `url(${PROJECTS_IMAGE_BASE}/${image})`,
+          backgroundPosition: 'center center',
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+        }}
+      ></div>
```

Comment:
이미지 로드 우선순위와 placeholder 기능 그리고 SEO를 위해서 

Next Image를 사용하며 key 값으로 다른 요소라는 것을 알려주는 방식으로 대응하면 어떨까요?!

```suggestion
      <Image
        key={title}
        src={`${PROJECTS_IMAGE_BASE}/${image}`}
        alt={title}
        fill
        priority
        quality={100}
        placeholder="blur"
        blurDataURL={`${PROJECTS_IMAGE_BASE}/${image}`}
        css={css`
          object-fit: cover;
        `}
      />
```

Reply:
next/image의 여러 속성을 조작해봤지만 잘 안되었는데 key prop 하나로 되는지는 몰랐네요..
감사합니다! 수정했습니다.